scylla/src/frame/value.rs

```diff
@@ -205,6 +244,19 @@ impl Value for String {
     }
 }
 
+impl Value for Vec<u8> {
+    fn serialize(&self, buf: &mut Vec<u8>) -> Result<(), ValueTooBig> {
+        let val_len: i32 = self.len().try_into().map_err(|_| ValueTooBig)?;
+        buf.put_i32(val_len);
+
+        for byte in self {
+            buf.put_u8(*byte);
+        }
```

Comment:
Use `extend_from_slice` here - it is more idiomatic and might have better performance:

```rust
buf.extend_from_slice(&self);
```

Reply:
Didn't think about it, makes sense! Fixed it.